docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+# v2 syntax
+version: '2'
+
+services:
+  # The database service
+  postgres:
+    image: postgres:9.6
+    container_name: jarbas-postgres
+    environment:
```

Comment:
Actually we need to improve that too, I'll spend more time on it when we get a infrastructure to work on.